README.md

```diff
@@ -732,6 +729,48 @@ manifest.ms = {
 };
 ```
 
+## Generating Icons
+
+Managing all the various icon sizes and types can be overwhelming. One solution is to start with a base image which can be 
```

Comment:
I would add a paragraph above this one to make it clear that you have to install this other addon first.

> ember-web-app doesn't generate icons or images. If you want to automate the generation of icons starting from a master image, you can install [ember-cli-image-transformer](https://github.com/jrjohnson/ember-cli-image-transformer).

What do you think? So people don't get the wrong idea that this addon installs ember-cli-image-transformer.

Reply:
I agree, that will be a lot clearer.